src/components/icons/indicator.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default ({ className }) => (
+  <svg
+    className={className}
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24">
+    <rect x="8" y="9" width="7.5" height="7.5" />
```

Comment:
![image](https://user-images.githubusercontent.com/1155816/91293198-9129d980-e78f-11ea-8579-bc98800ed844.png)

Overall I think this icon is pretty good, but could use some work to make it more visually consistent with the 👁️  icon. e.g.
- Make it white rather than blue?
- Use icon shape and not just opacity to indicate checked vs unchecked

cc @GabrielComymQB

Reply:
We've made some more changes here so please take a look now it's updated. Not the colour yet though, will need to see what @GabrielComymQB is thinking.

Reply:
Overall I think the new icons look good. Ideally I'd make them a little bigger, again so they're a comparable size/opacity to the 👁️ icon. Some design feedback:

- I really like the show/hide all icon. However I don't like that it often looks the same as the rest of the icons, so it's unclear that it behaves differently:
![image](https://user-images.githubusercontent.com/1155816/94673112-27609a80-030e-11eb-8f71-b227f110de39.png)
- I think the show/hide icon is a bit small, especially when it's showing hide all - it wasn't clear to me that it was a minus at first:
![image](https://user-images.githubusercontent.com/1155816/94673063-1021ad00-030e-11eb-9641-88bc6680e638.png)
- The icons are 100% opacity by default and have reduced opacity on hover. I'd have thought it should be the opposite, especially to be consistent with our other icons:
![image](https://user-images.githubusercontent.com/1155816/94673270-668eeb80-030e-11eb-8659-0d55c47e6059.png)
- I'd like there to be a tooltip on the show/hide icon, to indicate what will happen when you toggle it. This could be part of a separate PR. If you agree, let's add a ticket for it.

Reply:
I've updated the partial icon to be thinner and wider as per @GabrielComymQB's suggestion, I've also made the indicators brighter on hover by making them dimmer in their on state.